examples/stablecoin.py

```
@@ -0,0 +1,41 @@
+if msg['data'].startswith('mint:'):
+    # In a real contract, you would restrict this to an owner address!
+    # For this example, anyone can mint tokens to themselves.
+    amount = int(msg['data'].split(':')[1])
+    if amount <= 0:
+        raise Exception("Amount must be positive")
+    
+    sender = msg['sender']
+    storage[sender] = storage.get(sender, 0) + amount
+    storage['total_supply'] = storage.get('total_supply', 0) + amount
+
+elif msg['data'].startswith('transfer:'):
```

Ruff (0.15.15)

[error] Undefined name `msg`

(F821)